enhancements/workload-partitioning/pao-render.md

```diff
@@ -0,0 +1,190 @@
+   changes.
+1. **Keep all required headers.** If a section does not apply to an
+   enhancement, explain why but do not remove the section. This part
+   of the process is enforced by the linter CI job.
+
+The `Metadata` section above is intended to support the creation of tooling
+around the enhancement process.
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The Performance addon operator is a day 2 optional operator installed via OLM. However the [Workload partitioning](management-workload-partitioning.md) feature needs to have some configuration ready at the installation time when PAO is not yet ready.
+This feature is about adding a `render` mode to PAO that would allow an admin to pre-compute all the Openshift manifests that are needed in a way that prevents any typing mistakes from happening.
+These generated manifests can then be passed to the installer and later taken over by the PAO reconcile loop. 
+
+## Motivation
+
+There are many pieces that must be configured just right for the latency and resource management to work properly. PAO addresses that for day 2 latency tuning, but there was no way to use PAO's knowledge at the install time.
+
+### Goals
+
+1) PAO can execute outside of a cluster via podman/docker, consume user manifests from a directory and generate additional manifests to another (or the same) directory
+1) PAO takes all PerformanceProfile yamls from the input directory and generates all the usual manifests as yaml files to the output directory
+1) When a command line option `--enable-workload-partitioning` is passed to PAO in render mode, it also generates the necessary configuration for the `management` partition to work properly
+1) PAO at day 2 must not make any changes to the existing generated objects it owns
+1) PAO sets the ownership references on the generated yamls so the generated files are properly linked to the PerformanceProfiles
+
+### Non-Goals
+
+TBD
+
+## Proposal
+
+### User Stories
+
+- 1) [optional] The cluster administrator will collect PAO must-gather output from a test cluster with no partitioning enabled
+  1) [optional] The cluster administrator will execute the Performance profile creator to get well formed PerformanceProfiles
+  1) The PerformanceProfiles are placed into a directory
+  1) PAO is executed using podman run -v /input/dir:/input -v /output/dir:/output registry/performance-addon-operator render [optional --enable-workload-partitioning]
+  1) The files from both input and output directories are passed over to the installer
+
+### Implementation Details/Notes/Constraints [optional]
```

Comment:
Suggest we document the required inputs (performance profile(s), workload_partitioning CR) and the outputs manifests (kubeconfig, tuned etc). The performance profile CR will also need to be an output of running the render command.